CrisisTextLine/Elmer
# Sherlock word count tweak

#### What's this PR do?
The minimum word count for assignment short answers was set to 10, even though Mini Roleplay and Full Roleplay is stated to be 15. This fixes that.

#### Where should the reviewer start?
#### How should this be manually tested?
1. console.log out the `minWordCount` variable in the `formatHTML` function in `helpers.js`. 
2. Run `node app` in `www`.
3.
[full-roleplay.txt](https://github.com/CrisisTextLine/Elmer/files/725057/full-roleplay.txt)
[mini-roleplay.txt](https://github.com/CrisisTextLine/Elmer/files/725058/mini-roleplay.txt)
These are requests for the `canvas/typeformAssignment/Mini Roleplay` and `canvas/typeformAssignment/Full Roleplay` routes. Put these in Postman and submit.
4. `minWordCount` should equal 15 for either assignment.
#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/browse/AT-63
#### Questions:
